share/spice/public_holidays/public_holidays.js

```diff
@@ -0,0 +1,54 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_public_holidays = function(api_result) {
+
+        if ( !api_result || api_result.error || api_result.errors || !api_result.holidays || api_result.holidays.length === 0) {
+            return Spice.failed('public_holidays');
+        }
+
+        var script = $('[src*="/js/spice/public_holidays/"]')[0];
+        var source = $(script).attr("src");       
+        var year = decodeURIComponent(source.match(/public_holidays\/[a-z]{2}\/([0-9]{4})\/[^\/]+/)[1]);
+        var country = decodeURIComponent(source.match(/public_holidays\/[a-z]{2}\/[0-9]{4}\/([^\/]+)/)[1]);        
+        
+        if ( !year || !country ) {
+            return Spice.failed('public_holidays');                
+        }
+        
+        DDG.require('moment.js', function() {
+            Spice.add({
+                id: "public_holidays",            
+                name: "Answer",            
+                data: api_result,
+
+                meta: {
+                    sourceName: "timeanddate.com",
+                    sourceUrl: 'http://www.timeanddate.com/holidays/'
```

Comment:
This should link to a page related to the query, i.e. holidays for the searched country (and year if possible)